### DISCUSSION TITLE
How to specify other SVG conversion tools (Inkscape) into the chain

### DISCUSSION THREAD

**Discussion by fuhrmanator** [quarto-dev/quarto-cli#1140] [Q&A]
I'm using SVG a lot in my markdown, and it renders nicely (most of the time) in PDF by way of `rsvg-convert` part of the LaTeX chain:

> (rsvg-convert.exe:2516): Pango-WARNING **: couldn't load font "Helvetica Not-Rotated 13.640625", falling back to "Sans Not-Rotated 13.640625", expect ugly output.

I'd like to use Inkscape instead, as it does some things better, especially with fonts I've installed in Windows. 

Is there a way to specify that SVG conversions to PDF should be done using Inkscape?

**Comment by ajinkya-k** [comment]
I have the same question
